### DISCUSSION TITLE
Small Request: Integration Doesn't Show Logo on Integrations Page

### DISCUSSION THREAD

**Discussion by PablaV** [ufodone/envisalink_new#89] [Ideas]
Hey love the work done for this integration it works wayyy better than the original HA integration. Very small request/bug report but the custom integration doesn't show the Envisalink logo its just blank. Was wondering it that could be fixed in a future release. Again its super small and doesn't effect functionality just aesthetics. 

<img width="232" alt="image" src="https://github.com/ufodone/envisalink_new/assets/82848440/cf14f808-c09a-4ee5-92b3-a83a37a4cb16">
<img width="272" alt="image" src="https://github.com/ufodone/envisalink_new/assets/82848440/30370864-964c-4879-af9b-3a2ae50321bf">

**Comment by franquosino** [comment]
I would support that. Thanks for bringing it up. I thought about it, but knew with all the good work involved so far this would not be a priority on the list. Didn't have the guts to raise it :-)

**Comment by ufodone** [comment]
I have been putting off this change until I decide what to do about renaming the integration from `envisalink_new` to `envisalink`.  If I do the rename, the  logo should just start showing up.  If I decide against renaming, then I'll submit changes to the [brands](https://github.com/home-assistant/brands) repository.
